### ISSUE TITLE
Better asset copy jump fix

### ISSUE THREAD

**Issue by easyaspi314** [minetest/minetest#2127]
Instead of the Rect you are using, use 
```xml
<TextView
    ...
    android:layout_width="match_parent"
    android:ellipsize="middle" 
    android:singleLine="true"
    android:gravity="left"
    ...
    />
```

This will force the TextView to be the same width as the screen, and only be one line. It would look a bit like this:
Minetest/games/min...ame/mods/bucket
================---------------------

**Comment by sapier** [comment]
Hmm nice fix ... would've been way less work as the variant I did implement ... does it work on all android versions starting from 2.3? If yes I could replace the complex code I added.

**Comment by easyaspi314** [comment]
Yep. That code would work on it. Test it please.

**Comment by sapier** [comment]
merged 9c9b02ca8ad2753ad9c2e10e5b40e23029739634
